.github/workflows/check_db_migration_confict.yml

```diff
@@ -0,0 +1,59 @@
+name: Check DB migration conflict
+on:
+  push:
```

Comment:
Should this run only on pushes to `master` or is the intention to use this for release branches also?

Reply:
I was more thinking of feature branches, but this could be useful to release branches, too. We could potentially do some name matching here but since this workflow is relatively simple and fast, I think running on all branches is safer.